### PR TITLE
Awood/hanging indent

### PR DESCRIPTION
**Pull request by awood**
This fun little PR corrects line continuation indentation throughout our code base.  I've set Checkstyle to only allow one level of indentation on a continuation.  Annoyingly though, Checkstyle doesn't really do the right thing when it comes to nested method chains.  In those circumstances, I tagged the method `@SuppressWarnings("checkstyle:indentation")`.  My goal is to keep this particular check in enforcing mode for a week or so while everyone gets their editors configured accordingly.  Once everyone is set up, we can drop the check and the `SuppressWarnings` and just rely on everyone's editors to do the right thing for the most part.

### CODE REVIEW

File: server/src/main/java/org/candlepin/model/PoolCurator.java [hunk 1]
```diff
@@ -496,13 +494,11 @@ public Pool create(Pool entity) {
...
-            currentSession().createCriteria(Entitlement.class)
-                .setProjection(Projections.rowCount())
-                .createCriteria("pool")
-                    .createCriteria("sourceEntitlement")
-                        .add(Restrictions.idEq(entitlementId))
-                .list().get(0);
+        Integer result = (Integer) currentSession().createCriteria(Entitlement.class)
+            .setProjection(Projections.rowCount())
+            .createCriteria("pool")
+            .createCriteria("sourceEntitlement").add(Restrictions.idEq(entitlementId))
```
Comment (wottop) [inline server/src/main/java/org/candlepin/model/PoolCurator.java]: shouldn't the .add call be on a separate line? Its the same call level.

File: server/src/main/java/org/candlepin/pinsetter/tasks/HypervisorUpdateJob.java [hunk 2]
```diff
@@ -251,15 +252,14 @@ public void toExecute(JobExecutionContext context) throws JobExecutionException
                 }
                 // update reporter id if it changed
                 if (jobReporterId != null && reportedOnConsumer != null &&
-                        reportedOnConsumer.getHypervisorId() != null &&
-                        (reportedOnConsumer.getHypervisorId().getReporterId() == null || !jobReporterId
-                                .contentEquals(reportedOnConsumer.getHypervisorId().getReporterId()))) {
+                    reportedOnConsumer.getHypervisorId() != null &&
+                    (reportedOnConsumer.getHypervisorId().getReporterId() == null || !jobReporterId
+                            .contentEquals(reportedOnConsumer.getHypervisorId().getReporterId()))) {
```
Comment (wottop) [inline server/src/main/java/org/candlepin/pinsetter/tasks/HypervisorUpdateJob.java]: Still a double tab, maybe just one?

**Comment by awood** [comment]
@wottop fixed those issues you found

**Comment by wottop** [comment]
ack. will merge when tests are done.

**Comment by awood** [comment]
buildbot, retest this please

**Comment by wottop** [comment]
ack

**Pull request merged**